docs/upgrade/3.2.rst

```diff
@@ -26,29 +42,60 @@ Renaming cms_app, cms_toolbar, menu modules
 
 Old names are still supported but deprecated; support will be removed in 3.4.
 
+
 Action required
 ---------------
 
 In your own applications that use these modules, rename ``cms_app.py`` to ``cms_apps.py``,
 ``cms_toolbar.py`` to ``cms_toolbar.py`` and ``menu.py`` to ``menus.py``.
 
+
 New ApphookReloadMiddleware
 ===========================
 
-A new, optional middleware is available:
-``cms.middleware.utils.ApphookReloadMiddleware``. If used, this should be placed
-very near the top of your ``MIDDLEWARE`` tuple in settings. This middleware and
-other changes in the CMS for version 3.2 provides automatic reloading of URLs
-when changes to an apphook are made. This is tested to work in many types of
-production environments, but may not work in some others.
+Until now, changes to Apphooks have required a restart of the server in order to take effect. A new
+middleware class makes this automatic.
+
+The new middleware class, ``cms.middleware.utils.ApphookReloadMiddleware``, is optional. If used,
+this should be placed very near the top of your ``MIDDLEWARE_CLASSES`` tuple in settings. This
+middleware and other changes in the CMS for version 3.2 provide automatic reloading of URLs when
+changes to an apphook are made.
+
+.. note::
+
+   This has been tested and works in many production environments and deployment configurations,
+   but we haven't been able to test it with all possible set-ups. Please file an issue if you
+   discover one where it fails.
+
+
+Developer tools
+===============
+
+SASS-related changes
+--------------------
 
-*****************************
-Backward incompatible changes
-*****************************
+We now use `LibSass <https://github.com/sass/libsass>`_ rather than Compass for building static
+files (this only affects frontend developers *of* django CMS - contributors to it, not other users
+or developers). We've also adopted `CSSComb <http://csscomb.com>`_.
```

Comment:
we adopted it, but didn't actually include `.csscomb.json` to the repo, it's coming from https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/master/.csscomb.json